app/src/org/commcare/tasks/SimpleHttpTask.java

```diff
@@ -0,0 +1,108 @@
+package org.commcare.tasks;
+
+import android.content.Context;
+
+import org.commcare.CommCareApplication;
+import org.commcare.interfaces.HttpResponseProcessor;
+import org.commcare.interfaces.ResponseStreamAccessor;
+import org.commcare.network.ModernHttpRequester;
+import org.commcare.tasks.templates.CommCareTask;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.HashMap;
+
+/**
+ * Makes get/post request and delegates http response to receiver on completion
+ *
+ * @author Phillip Mates (pmates@dimagi.com)
+ */
+public class SimpleHttpTask
```

Comment:
Makes sense not to do in this PR, but is this something that it would make sense to start using in other places where we are doing server communication?